pom.xml

```diff
@@ -80,7 +80,8 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-api</artifactId>
-            <version>2.25</version>
+            <!-- Temp version until upstream PR released -->
```

Comment:
https://github.com/jenkinsci/workflow-api-plugin/pull/68

Reply:
In a perfect world, I prefer a link to the PR in the POM, but obviously that's not the end of the world. =)